controller/rest/model.go

```diff
@@ -63,6 +64,7 @@ type VolumeStats struct {
 	UsedLogicalBlocks string `json:"UsedLogicalBlocks"`
 	UsedBlocks        string `json:"UsedBlocks"`
 	SectorSize        string `json:"SectorSize"`
+	Size              string `json:"Size"`
```

Comment:
Have we updated the corresponding structure in maya ??

Reply:
yes